simple_app/app_simple.py

```diff
@@ -428,37 +404,24 @@ def search():
     """
     form = SearchForm()  # searchbar
     query = form.search.data  # the content in searchbar
+    if query is None:
+        query = ''
     db = SimpleDB(db_file, connection_arguments={'check_same_thread': False})  # open database
     results = db.search_object(query, fmt='astropy')  # get the results for that object
```

Comment:
You can specify `fmt='pandas'` (I think) to get a pandas DataFrame and then you don't need line 411